guides/v2.3/graphql/reference/quote-apply-giftcard.md

```diff
@@ -0,0 +1,76 @@
+---
+group: graphql
+title: applyGiftCardToCart mutation
+ee_only: True
+---
+
+The `applyGiftCardToCart` mutation applies a pre-defined gift card code to the specified cart.
+
+## Syntax
+
+ `mutation: applyGiftCardToCart(input: ApplyGiftCardToCartInput): ApplyGiftCardToCartOutput`
+
+## Example usage
+
+ The following example adds a gift card code called `048FQEGYUA73` to a cart.
+
+**Request**
+
+ ``` text
+mutation {
+   applyGiftCardToCart(
```

Comment:
The input should be
```
applyGiftCardToCart( input: {
      cart_id: "IeTUiU0oCXjm0uRqGCOuhQ2AuQatogjG"
      gift_card_code: "048FQEGYUA73"
   }
) 
```

Reply:
it is with input, we used wrappers everywhere